src/LoadingSpinner/loadingSpinner.scss

```diff
@@ -26,8 +26,8 @@ $thickness: 0.1em;
       -ms-transform: translateZ(0);
       -webkit-transform: translateZ(0);
       transform: translateZ(0);
-      margin-top: -$size / 2;
-      margin-left: -$size / 2;
+      margin-top: - calc($size / 2);
```

Comment:
just to be sure, is it working correctly? The minus before the `calc` and with a space in-between looks weird

Reply:
After some test this solution is not perfect. Close PR